src/Gateway/PayplugGateway.php

```diff
@@ -458,6 +459,14 @@ public function init_form_fields()
 		        'description' => '',
 		        'default'     => 'no',
 	        ],
+			'marketing'                   => [
```

Comment:
just a suggestion: change the name from this field to oney_product_animation or oney_product_marketing.